src/main/java/uk/gov/gchq/palisade/service/PolicyCacheWarmerFactory.java

```diff
@@ -43,9 +47,17 @@
     /**
      * Creates a {@link Policy} using the data within an implementation of the {@link PolicyCacheWarmerFactory}.
      *
-     * @return the {@link Policy} that has been created.
+     * @param users     a {@link List} of {@link UserCacheWarmerFactory} implementations
+     * @return          the {@link Policy} that has been created.
+     */
+    Entry<Resource, Policy> policyWarm(List<? extends UserCacheWarmerFactory> users);
+
+    /**
+     * Creates a {@link Resource} that will be passed to the Policy-Service
```

Comment:
Can you add more of a description of how this is the resource to which the policy will be applicable to

Reply:
Updated.